### PR TITLE
Removed logging from unit test

### PR DESCRIPTION
**Pull request by javidoc00**

### CODE REVIEW

File: include/omp/parallel_execution_omp.h [hunk 1]
```diff
@@ -1362,23 +1362,23 @@ void parallel_execution_omp::do_pipeline(
     for (;;) {
       auto item = input_queue.pop();
       if (!item.first) break;
-      std::cerr << "Processing: <" << *item.first << " , " << item.second << ">\n";
+      //std::cerr << "Processing: <" << *item.first << " , " << item.second << ">\n";
```
Comment (jdgarciauc3m) [inline include/omp/parallel_execution_omp.h]: This should not be commented but removed!

**Review by jdgarciauc3m — CHANGES_REQUESTED** [review]
Remove commented logging lines.

**Comment by javidoc00** [comment]
removed commented lines

**Review by jdgarciauc3m — APPROVED** [review]
OK. Thanks!

**Comment by jdgarciauc3m** [comment]
This solves issue #335

**Pull request merged**